core/deployment/src/main/java/io/quarkus/deployment/builditem/FeatureBuildItem.java

```diff
@@ -60,6 +60,7 @@
     public static final String UNDERTOW_WEBSOCKETS = "undertow-websockets";
     public static final String VERTX = "vertx";
     public static final String VERTX_WEB = "vertx-web";
+    public static final String SCALA = "scala";
```

Comment:
Ah and another minor additional comment: please respect the alphabetical order.

Reply:
Fixed it